ddl/db_test.go

```diff
@@ -3120,6 +3120,27 @@ func (s *testDBSuite1) TestModifyColumnCharset(c *C) {
 
 }
 
+func (s *testDBSuite1) TestSetTableFlashReplica(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use test_db")
+	s.mustExec(c, "drop table if exists t_flash;")
+	s.tk.MustExec("create table t_flash(a int, b int)")
+	defer s.mustExec(c, "drop table t_flash;")
+
+	t := s.testGetTable(c, "t_flash")
+	c.Assert(t.Meta().TiFlashReplica, IsNil)
+
+	s.tk.MustExec("alter table t_flash set tiflash replica 2 location labels 'a','b';")
```

Comment:
Looks that `alter table t_flash set tiflash replica 2` leads to empty location labels, is it OK?

Reply:
@lzmhhh123 PTAL

Reply:
It's OK. The labels for TiFlash are useless now. Many things need to change with importing the location concept. For now, we can ignore it.